### ISSUE TITLE
TypeError: Cannot read property 'filter' of undefined

### ISSUE THREAD

**Issue by zmni** [bmewburn/intelephense#22]
VSCode `v1.11.2`
Intelephense `v0.6.9`

```
[Error - 16.48.48] TypeError: Cannot read property 'filter' of undefined
    at SimpleVariableCompletion.completions (C:\Users\User\.vscode\extensions\bmewburn.vscode-intelephense-client-0.6.9\server\node_modules\intelephense\lib\completionProvider.js:319:40)
    at CompletionProvider.provideCompletions (C:\Users\User\.vscode\extensions\bmewburn.vscode-intelephense-client-0.6.9\server\node_modules\intelephense\lib\completionProvider.js:228:36)
    at Object.provideCompletions (C:\Users\User\.vscode\extensions\bmewburn.vscode-intelephense-client-0.6.9\server\node_modules\intelephense\lib\intelephense.js:107:35)
    at handleRequest (C:\Users\User\.vscode\extensions\bmewburn.vscode-intelephense-client-0.6.9\server\server.js:82:59)
    at handleRequest (C:\Users\User\.vscode\extensions\bmewburn.vscode-intelephense-client-0.6.9\server\server.js:155:17)
    at connection.onCompletion (C:\Users\User\.vscode\extensions\bmewburn.vscode-intelephense-client-0.6.9\server\server.js:81:16)
    at handleRequest (C:\Users\User\.vscode\extensions\bmewburn.vscode-intelephense-client-0.6.9\server\node_modules\vscode-jsonrpc\lib\main.js:237:37)
    at process.callback (C:\Users\User\.vscode\extensions\bmewburn.vscode-intelephense-client-0.6.9\server\node_modules\vscode-jsonrpc\lib\main.js:483:13)
    at emitTwo (events.js:106:13)
    at process.emit (events.js:191:7)
...
...
```

**Comment by bmewburn** [comment]
Thanks for the report. Was this within an anonymous function? If so it's fixed in the upcoming release #19

**Comment by zmni** [comment]
I missed that issue. Yes it's an anonymous function, writing the function directly on WP action hook. Same goes on #23, also on an anonymous function.
